From 1be953e2e3b25d573809900456cad2ed69f92f0c Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Sat, 20 Oct 2018 07:21:10 +0000
Subject: [PATCH 1/7] arm: dts: sunxi: H3 NanoPi - add missing ethernet0 alias

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/arch/arm/dts/sun8i-h3-nanopi.dtsi b/arch/arm/dts/sun8i-h3-nanopi.dtsi
index c7c3e7d8b3..842beb4705 100644
--- a/arch/arm/dts/sun8i-h3-nanopi.dtsi
+++ b/arch/arm/dts/sun8i-h3-nanopi.dtsi
@@ -51,6 +51,7 @@
 / {
 	aliases {
 		serial0 = &uart0;
+		ethernet0 = &emac;
 	};
 
 	chosen {
