From ceac51446669c9efa49f4f85f73283b635aabcb7 Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Tue, 23 Jan 2018 11:52:20 +0100
Subject: [PATCH 3/9] ARM: dts: sun4i: Correct pinctrl nodes

spi2_cs0_pc_pins -> spi2_cs0_pc_pin

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/arch/arm/boot/dts/sun4i-a10.dtsi b/arch/arm/boot/dts/sun4i-a10.dtsi
index e88daa4ef1af..2450498eacfc 100644
--- a/arch/arm/boot/dts/sun4i-a10.dtsi
+++ b/arch/arm/boot/dts/sun4i-a10.dtsi
@@ -779,7 +779,7 @@
 				function = "spi2";
 			};
 
-			spi2_cs0_pc_pins: spi2-cs0-pc-pin {
+			spi2_cs0_pc_pin: spi2-cs0-pc-pin {
 				pins = "PC19";
 				function = "spi2";
 			};
-- 
2.23.0

