From 495f0e93ccd704f41deab17ff215f13a4a9ef1d2 Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Tue, 23 Jan 2018 11:52:20 +0100
Subject: [PATCH 3/9] ARM: dts: sun4i: Correct pinctrl nodes

spi2_cs0_pc_pins -> spi2_cs0_pc_pin

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/arch/arm/boot/dts/sun4i-a10.dtsi b/arch/arm/boot/dts/sun4i-a10.dtsi
index 4c268b70b735..f3fd0779d346 100644
--- a/arch/arm/boot/dts/sun4i-a10.dtsi
+++ b/arch/arm/boot/dts/sun4i-a10.dtsi
@@ -783,7 +783,7 @@
 				function = "spi2";
 			};
 
-			spi2_cs0_pc_pins: spi2-cs0-pc-pin {
+			spi2_cs0_pc_pin: spi2-cs0-pc-pin {
 				pins = "PC19";
 				function = "spi2";
 			};
-- 
2.24.0

