From 8500098a4d4fc57748b3245a424381096c72e0e4 Mon Sep 17 00:00:00 2001
From: Edmunt Pienkowsky <roed@onet.eu>
Date: Tue, 23 Jan 2018 11:52:20 +0100
Subject: [PATCH 3/9] ARM: dts: sun4i: Correct pinctrl nodes

spi2_cs0_pc_pins -> spi2_cs0_pc_pin

Signed-off-by: Edmunt Pienkowsky <roed@onet.eu>

diff --git a/arch/arm/boot/dts/sun4i-a10.dtsi b/arch/arm/boot/dts/sun4i-a10.dtsi
index 73c3ac42095f..0e8b3db168ca 100644
--- a/arch/arm/boot/dts/sun4i-a10.dtsi
+++ b/arch/arm/boot/dts/sun4i-a10.dtsi
@@ -780,7 +780,7 @@
 				function = "spi2";
 			};
 
-			spi2_cs0_pc_pins: spi2-cs0-pc-pin {
+			spi2_cs0_pc_pin: spi2-cs0-pc-pin {
 				pins = "PC19";
 				function = "spi2";
 			};
-- 
2.21.0

